boa3_test/test_drive/model/network/payloads/testtransaction.py

```diff
@@ -55,7 +56,17 @@ def from_json(cls, json: Dict[str, Any]) -> TestTransaction:
         tx = object.__new__(cls)  # init was causing errors with inherited classes
         tx._hash = tx_hash
         tx._script = script
+
         tx._attributes = []
+        if 'attributes' in json:
+            attributes_json = json['attributes']
+
+            for attribute in attributes_json:
+                from boa3_test.test_drive.model.network.payloads.transactionattribute import \
+                    TransactionAttributeType
+                if attribute['type'] == TransactionAttributeType.ORACLE_RESPONSE:
+                    from boa3_test.test_drive.model.network.payloads.oracleresponse import OracleResponse
+                    tx._attributes.append(OracleResponse.from_json(attribute))
```

Comment:
I think it's better to move this logic to TransactionAttribute and import there the method from the respective classes to deserialize the given json. Something like the following:

```suggestion
        if 'attributes' in json:
            tx._attributes = [TransactionAttribute.from_json(atttribute) for attribute in json['attributes']]
        else:
            tx._attributes = []
```